docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Clustering/KMeans.cs

```diff
@@ -9,7 +9,7 @@ public static void Example()
         {
             // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
             // as well as the source of randomness.
-            var ml = new MLContext();
+            var ml = new MLContext(seed: 1, conc: 1);
```

Comment:
>seed: 1, conc: 1 [](start = 35, length = 16)

is this needed?

Reply:
Yes, its needed otherwise results different on every run.

---
In reply to: [262274167](https://github.com/dotnet/machinelearning/pull/2819#discussion_r262274167) [](ancestors = 262274167)